tkg/client/update_credentials.go

```diff
@@ -197,3 +214,98 @@ func (c *TkgClient) updateVSphereCredentialsForCluster(clusterClient clusterclie
 
 	return nil
 }
+
+// UpdateAzureClusterCredentials update azure cluster credentials
+func (c *TkgClient) UpdateAzureClusterCredentials(clusterClient clusterclient.Client, options *UpdateCredentialsOptions) error {
+	if options.AzureUpdateClusterOptions.AzureTenantID == "" || options.AzureUpdateClusterOptions.AzureClientID == "" || options.AzureUpdateClusterOptions.AzureClientSecret == "" {
```

Comment:
The user still needs to provide 3 parts of information if the user just wants to update the client secret, correct? If so, it this the same as vSphere credential update?

Reply:
Yes. It followed the same login as vsphere.